scripts/run_e2e_tests.sh

```diff
@@ -40,14 +40,12 @@
 
 function cleanup {
   # Send a kill signal to the dev server.
-  #
+  kill `ps aux | grep "[Dd]ev_appserver.py --host=0.0.0.0 --port=9001" | awk '{print $2}'`
```

Comment:
Good question. While debugging, I kept running into issues with the dev server not shutting down correctly before the selenium server shut down. So I switched the order and removed an unnecessary comment.